tljh/configurer.py

```diff
@@ -191,6 +203,43 @@ def update_traefik_api(c, config):
     c.TraefikTomlProxy.traefik_api_password = config['traefik_api']['password']
 
 
+def set_cull_idle_service(c, config):
+    """
+    Set Idle Culler service
+    """
+    cull_cmd = [
+       sys.executable, '/srv/src/tljh/cull_idle_servers.py'
+    ]
+    if config['services']['cull']['timeout']:
```

Comment:
Since this was set in the default, I *think* these properties will always be present even if the user doesn't override them, so you don't need all the conditionals.

Reply:
Thanks a lot for your feedback @yuvipanda :heart: 
One small question: are the default values ok? Should I change anything there?

Reply:
yw! Thanks for working on this :) the defaults look ok to me as long as we document them.